automotive/src/main/java/au/com/shiftyjelly/pocketcasts/AutomotiveAboutFragment.kt

```diff
@@ -46,18 +44,16 @@ import au.com.shiftyjelly.pocketcasts.ui.R as UR
 @AndroidEntryPoint
 class AutomotiveAboutFragment : Fragment() {
 
-    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
-        return ComposeView(requireContext()).apply {
-            setContent {
-                AutomotiveTheme {
-                    AboutPage(
-                        onOpenLicenses = { openLicenses() },
-                        onOpenLogs = { onOpenLogs() },
-                        onOpenUrl = { openUrl(it) },
-                    )
-                }
-            }
-        }
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?,
+    ) = content {
+        AboutPage(
```

Comment:
Why don't we wrap it into AutomotiveTheme? 🤔

Reply:
Fixed in https://github.com/Automattic/pocket-casts-android/pull/3229/commits/08e2e8ba199fc30429bc60323236cd7e0dba8f13